### PR TITLE
UICHKIN-276: Remove undefined barcode

### PR DESCRIPTION
**Pull request by annamelnyk**
## Purpose
Undefined Bar Code - Requester info not showing

## Refs
https://issues.folio.org/browse/UICHKIN-276

## Screenshots
**Before**
image

**After**
image

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by zburke — CHANGES_REQUESTED** [review]
The change looks good; please update the [`src/util.test.js#convertToSlipData()`](https://github.com/folio-org/ui-checkin/blob/968a5fc1aae1557c3aefdafa2a5d15f78f1cbe13/src/util.test.js#L54) test to handle this condition.

**Comment by annamelnyk** [comment]
> The change looks good; please update the [`src/util.test.js#convertToSlipData()`](https://github.com/folio-org/ui-checkin/blob/968a5fc1aae1557c3aefdafa2a5d15f78f1cbe13/src/util.test.js#L54) test to handle this condition.

I didn't see this file. Thanks! Done.

File: src/util.test.js [hunk 1]
```diff
@@ -109,6 +109,7 @@ describe('convertToSlipData', () => {
     };
 
     const o = convertToSlipData(source, intl, tz, locale, 'Chicken');
+    const requesterBarcode = source.requester.barcode ? `<Barcode>${source.requester.barcode}</Barcode>` : '';
```
Comment (zburke) [inline src/util.test.js:112]: Should we be testing both sides of this ternary?

**Review by zburke — APPROVED** [review]
❤️ 🚀

**Review by mkuklis — APPROVED** [review]

**Pull request merged**